frontend/src/static/js/components/webstatus-overview-filters.ts

```diff
@@ -229,11 +230,14 @@ export class WebstatusOverviewFilters extends LitElement {
   };
 
   gotoFilterQueryString(): void {
-    const newUrl = formatOverviewPageUrl(this.location, {
-      q: (this.typeaheadRef.value as WebstatusTypeahead).value,
-      start: 0,
-    });
-    window.location.href = newUrl;
+    const newUrl = `${window.location.origin}${formatOverviewPageUrl(
+      this.location,
+      {
+        q: (this.typeaheadRef.value as WebstatusTypeahead).value,
+        start: 0,
+      },
+    )}`;
```

Comment:
I want to remove as many references from window.location from our components and leave them for the utils library. So let's revert this.

```suggestion
    const newUrl = formatOverviewPageUrl(this.location, {
      q: (this.typeaheadRef.value as WebstatusTypeahead).value,
      start: 0,
    });
```